docker/test/develop.yml

```diff
@@ -0,0 +1,42 @@
+# This needs at least compose 1.6.0
+version: '2'
+
+services:
+  postgresdata:
+    image: postgres:9.5
+    command: /bin/true
+
+  postgres:
+    image: postgres:9.5
+    volumes_from:
+      - postgresdata
+    environment:
+      POSTGRES_PASSWORD: myhuginnpassword
+      POSTGRES_USER: huginn
+
+  huginn_test_postgres:
+    build:
+      context: ../../
+      dockerfile: docker/test/Dockerfile
+    restart: always
+    extends:
```

Comment:
you're right there isn't really any reuse happening so the `extends` isn't necessary - will remove